### ISSUE TITLE
Mythics Gear.

### ISSUE THREAD

**Issue by neano300** [Valhalla-Gaming/Tracker#8740]
**Describe the issue you're having**:
The green items convert to mythics when take it.
**Explain how you expect it work**:
the green items  dont will convert to mythics.
**Steps to reproduce the problem**:

1.  Kill a mob 
2. loot any item
3.  Mythic.

**Links to Wowhead, YouTube, etc**:
Pohotos: 
http://imgur.com/PeEC8LP
http://imgur.com/EgzrekZ

**Comment by lavalampe77** [comment]
Of course, every green item should normally have a chance be be a better quality then green what means rare,epic,mythic or titan forged.
This should also happen outside of dungeons.

**Comment by Maldazar-Valhalla** [comment]
It has chance to become warforged or titanforged, or get an additional stat, it can't become mythic.

**Comment by Analog-Valhalla** [comment]
Currently this is only a visual bug. Loot adjustments need to be made aswell.
